terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up.ts

```diff
@@ -85,11 +85,9 @@ export async function scaleUp(
       continue;
     }
 
-    if (runnerType.runnerTypeName.includes('linux.2xlarge')) {
-      if ((await getExperimentValue('stresstest_ignorereq', 0)) > Math.random() * 100) {
-        console.warn(`Stresstest ignore request for scale ${runnerType.runnerTypeName}`);
-        continue;
-      }
+    if (await getJoinedStressTestExperiment('stresstest_ignorereq', runnerType.runnerTypeName)) {
+      console.warn(`Stresstest stresstest_ignorereq: ignore request for scale ${runnerType.runnerTypeName}`);
```

Comment:
idea for clearer logs

```suggestion
      console.warn(`Stresstest stresstest_ignorereq active: ignoring request for scale ${runnerType.runnerTypeName}`);
```